src/components/pages/my-space.js

```diff
@@ -48,7 +47,7 @@ function MySpace({ initialState, PageTemplate }) {
   };
 
   const alerts = getGlobalAlerts(pageName, user);
-  const isRoomsTabEnabled = !clientConfig.disabledFeatures.includes(FEATURE_TOGGLES.rooms);
+  const isRoomsTabEnabled = clientConfig.areRoomsEnabled;
```

Comment:
I guess this can be inlined now.

Reply:
Done, I've also checked for other occurrences and there seems to be no more indirection in the code.